build.sbt

```diff
@@ -10,6 +10,7 @@ val kindProjectorVersion = "0.13.2"
 
 ThisBuild / Test / bspEnabled                                        := false
 ThisBuild / ScalafixConfig / bspEnabled.withRank(KeyRanks.Invisible) := false
+ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
```

Comment:
Isn't this set by `sbt-lucuma` ?

Reply:
It should be but it isn't. Something changed as it was broken even before